sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -265,6 +270,10 @@ Job pollJob(
             return job;
           }
           // The job is not DONE, wait longer and retry.
+          if (Instant.now().isAfter(nextLog)) {
+            LOG.info("Still waiting for BigQuery job {}", jobRef.getJobId());
+            nextLog = Instant.now().plus(POLLING_LOG_GAP);
```

Comment:
If you're up for it, would also be good to plumb something like this into everywhere we print a starting or a "still waiting for" message:

```
bq show -j --format=prettyjson --project_id=google.com:deft-testing-load dataflow_job_1331623617330908623
```